nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/pom.xml

```diff
@@ -27,7 +27,7 @@
     <version>2.0.0-SNAPSHOT</version>
     <packaging>war</packaging>
     <properties>
-        <node.version>v20.5.1</node.version>
+        <node.version>v21.6.1</node.version>
```

Comment:
FYI:

'''
nifi git:(PR-8410) npm run start

> nifi-web-frontend@0.0.0 start
> ng serve

Node.js version v21.6.1 detected.
Odd numbered Node.js versions will not enter LTS status and should not be used for production. For more information, please see https://nodejs.org/en/about/previous-releases/.
'''

Reply:
https://issues.apache.org/jira/browse/NIFI-12802